example/src/main/scala/FileStreaming.scala

```diff
@@ -19,5 +19,5 @@ object FileStreaming extends App {
 
   // Run it like any simple app
   override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] =
-    Server.start(8090, app.silent).exitCode
+    Server.start(8090, HttpApp(app.asHttp.silent)).exitCode
```

Comment:
```suggestion
    Server.start(8090, app.silent).exitCode
```

- [ ] Move `silent` to HttpApp